addons/hw_drivers/iot_handlers/drivers/KeyboardUSBDriver_L.py

```diff
@@ -188,15 +188,14 @@ def save_layout(self, layout):
         else:
             data = {}
         data[self.device_identifier] = layout
-        helpers.write_file('odoo-keyboard-layouts.conf', json.dumps(data))
+        helpers.update_config({'keyboard-layouts', data})
```

Comment:
since drivers are updated on every Odoo restart, but helpers are not, this can break some IoT Boxes in v17